packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -243,31 +243,93 @@ export function getProfile(node: vscode.TreeItem) {
 export async function readConfigFromDisk() {
     const mProfileInfo = await getProfileInfo(globals.ISTHEIA);
     let rootPath;
-    if (vscode.workspace.workspaceFolders) {
-        rootPath = vscode.workspace.workspaceFolders[0].uri.fsPath;
-        await mProfileInfo.readProfilesFromDisk({ projectDir: getFullPath(rootPath) });
-    } else {
-        await mProfileInfo.readProfilesFromDisk({ homeDir: getZoweDir() });
+    try {
+        if (vscode.workspace.workspaceFolders) {
+            rootPath = vscode.workspace.workspaceFolders[0].uri.fsPath;
+            await mProfileInfo.readProfilesFromDisk({ projectDir: getFullPath(rootPath) });
+        } else {
+            await mProfileInfo.readProfilesFromDisk({ homeDir: getZoweDir() });
+        }
+        if (mProfileInfo.usingTeamConfig) {
+            globals.setConfigPath(rootPath);
+            globals.LOG.debug(
+                'Zowe Explorer is using the team configuration file "%s"',
+                mProfileInfo.getTeamConfig().configName
+            );
+            const layers = mProfileInfo.getTeamConfig().layers || [];
+            const layerSummary = layers.map(
+                (config: imperative.IConfigLayer) =>
+                    `Path: ${config.path}: ${
+                        config.exists
+                            ? "Found, with the following defaults:" +
+                              JSON.stringify(config.properties?.defaults || "Undefined default")
+                            : "Not available"
+                    } `
+            );
+            globals.LOG.debug(
+                "Summary of team configuration files considered for Zowe Explorer: %s",
+                JSON.stringify(layerSummary)
+            );
+        }
+    } catch (error) {
+        if (error.message.toString().includes("Error parsing JSON")) {
```

Comment:
Could there be another profile error that does not result in this error message? Looks like if error message is different, Zowe Explorer will fail to activate. Checking for error message strings is not a robust approach.

Reply:
I think if Imperative fails to read profile for any reason it should still activate, and open the file if it exists.

Reply:
this is already handled whether it is a parsing error or an fs error the error will be caught and message shown continuing with activation.  Only with parsing error the config file in question will be opened